### PR TITLE
Use installer instead of building from source in the Tutorial

### PR DESCRIPTION
**Pull request by jsturtevant**
Signed-off-by: James Sturtevant <jstur@microsoft.com>

## Description
updates docs to address not needing to build the project to use the eBPF. see https://github.com/microsoft/ebpf-for-windows/issues/1408 and https://github.com/microsoft/ebpf-for-windows/issues/1405

Adds a section with tips for installing ebpf programs with pointers where to learn more.

_Describe the purpose of and changes within this Pull Request._

## Testing

_Do any existing tests cover this change? Are new tests needed?_

## Documentation

_Is there any documentation impact for this change?_

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -690,8 +662,10 @@ int func()
 Let's compile it and see what it looks like.   Here we compile with `-g`
 to include source line info:
 
+> note: Replace `.\eBPF-for-Windows.0.4.0\build\native\include\` with the path to the ebpf-for-windows nuget `./include` directory (should be in directory where you ran `nuget install eBPF-for-Windows`). This is the first time we included header files while building so we need to use the Windows eBPF headers which we get from the nuget package.
```
Comment (dthaler) [inline docs/tutorial.md:665]:
```suggestion
> note: Replace `.\eBPF-for-Windows.0.4.0\build\native\include\` with the path to the ebpf-for-windows nuget `./include` directory (should be in the directory where you ran `nuget install eBPF-for-Windows`). This is the first time we included header files while building so we need to use the Windows eBPF headers which we get from the nuget package.
```

File: docs/tutorial.md [hunk 2]
```diff
@@ -956,7 +930,35 @@ FDs or addresses.  When the program is actually installed, the relocation
 section will be used to insert the actual map address into the executable
 code.
 
+## 5. Installing eBPF programs
+
+The `netsh ebpf` can also install your eBFP program. As an example, the program we created above won't do much but we can install it via:
```
Comment (dthaler) [inline docs/tutorial.md:935]:
```suggestion
`netsh ebpf` can also install your eBPF program. As an example, the program we created above won't do much but we can install it via:
```

File: docs/tutorial.md [hunk 3]
```diff
@@ -956,7 +930,35 @@ FDs or addresses.  When the program is actually installed, the relocation
...
+
+```
+netsh ebpf add program .\myxdp.o xdp
+```
+
+See it installed:
```
Comment (dthaler) [inline docs/tutorial.md:941]:
```suggestion
To see it installed:
```

File: docs/tutorial.md [hunk 4]
```diff
@@ -956,7 +930,35 @@ FDs or addresses.  When the program is actually installed, the relocation
...
+
+```
+> netsh ebpf show programs
+
+    ID  Pins  Links  Mode       Type           Name
+======  ====  =====  =========  =============  ====================
+ 65568     1      1  JIT        xdp            my_xdp_parser
+```
+
+And remove it by the id
```
Comment (dthaler) [inline docs/tutorial.md:951]:
```suggestion
And remove it by the id, which we saw above is 65568:
```

File: docs/tutorial.md [hunk 5]
```diff
@@ -956,7 +930,35 @@ FDs or addresses.  When the program is actually installed, the relocation
...
+
+```
+> netsh ebpf delete program <id>
```
Comment (dthaler) [inline docs/tutorial.md:954]:
```suggestion
> netsh ebpf delete program 65568
```

File: docs/tutorial.md [hunk 6]
```diff
@@ -956,7 +930,35 @@ FDs or addresses.  When the program is actually installed, the relocation
...
+```
+
+You can also do install and interact with the eBPF programs programmatically. See https://github.com/microsoft/ebpf-for-windows-demo for examples.
```
Comment (dthaler) [inline docs/tutorial.md:957]:
```suggestion
You can also install and interact with the eBPF programs programmatically. See https://github.com/microsoft/ebpf-for-windows-demo for examples.
```

File: docs/tutorial.md [hunk 7]
```diff
@@ -956,7 +930,35 @@ FDs or addresses.  When the program is actually installed, the relocation
...
+
+Learn more at about using eBPF at https://github.com/microsoft/ebpf-for-windows/blob/main/docs/GettingStarted.md#using-ebpf-for-windows
```
Comment (dthaler) [inline docs/tutorial.md:959]:
```suggestion
Learn more about using eBPF at https://github.com/microsoft/ebpf-for-windows/blob/main/docs/GettingStarted.md#using-ebpf-for-windows
```

**Review by dthaler — COMMENTED** [review]
Thanks for this contribution!  See grammatical edits in comments.

File: docs/tutorial.md [hunk 8]
```diff
@@ -3,7 +3,14 @@
 This tutorial illustrates how eBPF works and in particular how the eBPF verifier works on Windows,
 starting from authoring a new eBPF program in C.
 
-To try out this tutorial yourself, you should first install the [Prerequisites](GettingStarted.md#Prerequisites).
+To try out this tutorial yourself, you will need:
+
+- Clang and nuget from [Prerequisites tools](GettingStarted.md#Prerequisites). 
+- a VM that can [load a Windows driver](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/GettingStarted.md#installing-ebpf-for-windows). 
+  - Follow in the [VM install instructions](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/vm-setup.md) to get started quickly.
```
Comment (dthaler) [inline docs/tutorial.md:10]:
```suggestion
  - Follow the [VM install instructions](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/vm-setup.md) to get started quickly.  
```

File: docs/tutorial.md [hunk 9]
```diff
@@ -690,8 +662,10 @@ int func()
 Let's compile it and see what it looks like.   Here we compile with `-g`
 to include source line info:
 
+> note: Replace `.\eBPF-for-Windows.0.4.0\build\native\include\` with the path to the ebpf-for-windows nuget `./include` directory (should be in the directory where you ran `nuget install eBPF-for-Windows`). This is the first time we included header files while building so we need to use the Windows eBPF headers which we get from the nuget package.
```
Comment (dthaler) [inline docs/tutorial.md:665]:
```suggestion
> Note: Replace `.\eBPF-for-Windows.0.4.0\build\native\include\` with the path to the ebpf-for-windows nuget `./include` directory (should be in the directory where you ran `nuget install eBPF-for-Windows`). This is the first time we included header files while building so we need to use the Windows eBPF headers which we get from the nuget package.
```

**Review by dthaler — DISMISSED** [review]

**Review by dthaler — APPROVED** [review]

File: docs/tutorial.md [hunk 10]
```diff
@@ -690,8 +662,10 @@ int func()
 Let's compile it and see what it looks like.   Here we compile with `-g`
 to include source line info:
 
+> Note: Replace `.\eBPF-for-Windows.0.4.0\build\native\include\` with the path to the ebpf-for-windows nuget `./include` directory (should be in the directory where you ran `nuget install eBPF-for-Windows`). This is the first time we included header files while building so we need to use the Windows eBPF headers which we get from the nuget package.
+
 ```
-> clang -I ../../include -target bpf -Werror -g -O2 -c helpers.c -o helpers.o
+> clang -I .\eBPF-for-Windows.0.4.0\build\native\include\ -target bpf -Werror -g -O2 -c helpers.c -o helpers.o
```
Comment (dthaler) [inline docs/tutorial.md:668]: This line will need to be updated every time we do another release.  Should add it to the docs\ReleaseProcess.md document and @Alan-Jowett should update #1322 accordingly

**Review by Alan-Jowett — APPROVED** [review]

**Pull request merged**